ts/components/dialog/network/NodeImage.tsx

```diff
@@ -42,39 +42,33 @@ type Props = {
   width: string;
   height: string;
   loading?: boolean;
-  error?: Error;
 };
 
-export const NodeImage = (props: Props) => {
-  const { count, width, height, loading, error } = props;
-  const isOnline = useIsOnline();
-
-  if (error) {
-    window.log.error(`[NodeImage] node count error: ${error}`);
-  }
+const nodeComps: Record<number, (props: NodeGraphProps) => JSX.Element> = {
+  1: NodeGraph1,
+  2: NodeGraph2,
+  3: NodeGraph3,
+  4: NodeGraph4,
+  5: NodeGraph5,
+  6: NodeGraph6,
+  7: NodeGraph7,
+  8: NodeGraph8,
+  9: NodeGraph9,
+  10: NodeGraph10,
+};
 
-  const nodeComps: Record<number, (props: NodeGraphProps) => JSX.Element> = {
-    1: NodeGraph1,
-    2: NodeGraph2,
-    3: NodeGraph3,
-    4: NodeGraph4,
-    5: NodeGraph5,
-    6: NodeGraph6,
-    7: NodeGraph7,
-    8: NodeGraph8,
-    9: NodeGraph9,
-    10: NodeGraph10,
-  };
+export const NodeImage = ({ width, height, loading }: Props) => {
+  const { swarmNodeCount } = useSecuringNodesCount();
 
-  const NodeComp = nodeComps[count];
+  const NodeComp = nodeComps[swarmNodeCount ?? 0];
```

Comment:
There is no `nodeComps[0]`, so this would be undefined right? could be better to use something like 7 (the goal swarm size)?

Reply:
this is ok as we want the loader to be shown in this case